CCS/3. Tutorials/6. 使用 TensorFlow 訓練 MNIST 手寫數字辨識模型.md

```diff
@@ -1 +1,66 @@
+---
+title: MNIST
+tags: HowTo, CCS, Interactive, TW
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+
+# HowTo：Using TensorFlow to train MNIST handwriting digital recognition model
+
+
+The following instruction is how to build an Interactive Container in TWCC and using Jupyter Notebook's working environment to carry out MNIST (Handwriting Digital Recognition Data Set) AI training
+
+### Step 1. Sign in TWCC
+
+- If you don’t have an account yet, please refer to [Register for an TWCC account](https://www.twcc.ai/doc?page=register_account).
+
+### Step 2. Create a TWCC Interactive Container
+
+- Please refer to [Interactive Container](https://www.twcc.ai/doc?page=container#建立開發型容器) to create a Interactive Container.
+
+- Please select TensorFlow for the image file type, select the version that supports Python 3 for the image file, and set the hardware to support 1 GPU.
```

Comment:
[SUG]

image file type ...GPU. > 

image type, select an image version that supports Python 3, and select 1 GPU for the hardware.